python/cuml/test/dask/test_label_binarizer.py

```diff
@@ -32,10 +32,13 @@ def test_basic_functions(labels, client):
     fit_labels, xform_labels = labels
 
     s = cp.asarray(fit_labels, dtype=np.int32)
-    df = dask.array.from_array(s)
+    df = dask.array.from_array(s).rechunk((1,))
```

Comment:
Do we also need a multi-chunk test? I could imagine both are helpful to have?

Reply:
This is actually a multi-chunk test. It sets the chunk size to 1, to force the number of chunks to be equal to the number of elements (the worst-case).

Reply:
Great. Let's push this in with the fix for 0.17 then add a single-chunk test for comparison in 0.18